workloads/hyper-scale/common.sh

```diff
@@ -1,25 +1,42 @@
 #!/usr/bin/env bash
+set -x
 
 source env.sh
 
 prep(){
     if [[ -z $(go version) ]]; then
-        curl -L https://go.dev/dl/go1.17.6.linux-amd64.tar.gz -o go1.17.6.linux-amd64.tar.gz
-        tar -C /usr/local -xzf go1.17.6.linux-amd64.tar.gz
+        curl -L https://go.dev/dl/go1.18.2.linux-amd64.tar.gz -o go1.18.2.linux-amd64.tar.gz
+        tar -C /usr/local -xzf go1.18.2.linux-amd64.tar.gz
         export PATH=$PATH:/usr/local/go/bin
-        git clone --branch main https://github.com/openshift/hypershift
+    fi
+    if [[ ${HYPERSHIFT_CLI_INSTALL} != "false" ]]; then
+        echo "Remove current Hypershift CLI directory.."
+        sudo rm -rf hypershift || true
+        sudo rm /usr/local/bin/hypershift || true
```

Comment:
Can we have it use a local version of hypershift as to a) not cause conflicts if I am running two tests from the same host and b) not explode my in use copy of it if I am testing something just for this e2e run?

Reply:
Yes, That is possible by setting  `HYPERSHIFT_CLI_INSTALL` variable to `false`, so it will not re-install the CLI and use the local version.

Reply:
Per my slack message. To clarify, I did not mean so it uses my currently installed hypershift cli but copies down to a local/dynamic folder. ie if i'm running it from /home/dry/foo it creates /home/dry/foo/hypershift-cli/...

Reply:
Sorry to jump in between, but maybe like this, to avoid conflicts w/ local clones of hypershift.

```
export HYPERSHIFT_GIT_REPO=`mktemp -d`
cd $HYPERSHIFT_GIT_REPO
git clone -q --depth=1 --single-branch --branch ${HYPERSHIFT_CLI_VERSION} ${HYPERSHIFT_CLI_FORK} 
pushd hypershift
.....
```

Reply:
yes, thats right. @dry923 lets do it in a separate PR? we can workaround that by setting the cli install to `false` for now.